hyperspy/tests/learn/test_svd_pca.py

```diff
@@ -149,6 +154,8 @@ def test_solver_auto(self):
         normX = np.linalg.norm(X - Y)
         assert normX < self.tol
 
+    @pytest.mark.skipif(LooseVersion(scipy.__version__) < LooseVersion("1.4.0"),
```

Comment:
why not `pytest.importorskip("scipy", min_version="1.4.0")` inside the test

Reply:
Done! I wasn't aware of this one!